packages/compiler/test/server/completion.test.ts

```diff
@@ -16,19 +16,34 @@ describe("compiler: server: completion", () => {
       }
       `
     );
-
     check(completions, [
       {
-        label: "int32",
-        insertText: "int32",
-        kind: CompletionItemKind.Keyword,
+        label: "Cadl",
         documentation: undefined,
+        kind: 9,
+        detail: "intrinsic Z:/test/.cadl/dist/lib/decorators.js.Cadl",
```

Comment:
```suggestion
        detail: "intrinsic Z:/test/.cadl/dist/lib/decorators.js.Cadl",
```
Should probably not include the file path in the fully qualified name. That here also would fail the CI on linux.